services/ui-src/src/components/export/ExportedModalDrawerReportSection.tsx

```diff
@@ -1,15 +1,279 @@
 // components
 import { Box, Heading } from "@chakra-ui/react";
+import { Table } from "components";
 // utils
+import { useEffect, useState } from "react";
 import { useStore } from "utils";
-import { ModalDrawerReportPageShape } from "types";
+import { ModalDrawerReportPageShape, AnyObject } from "types";
+import _ from "lodash";
 
 export const ExportedModalDrawerReportSection = ({
   section: { entityType, verbiage },
 }: Props) => {
+  const [overflow, setOverflow] = useState(false);
   const { report } = useStore() ?? {};
   const entities = report?.fieldData?.[entityType];
 
+  const truncateHeader = (text: string) => {
+    return _.truncate(text, { length: 30 });
+  };
+
+  // if Transition Benchmark Header title has an abbrev.just display that
+  const getTableHeaders = () => {
+    let headers = [];
+    const quarterHeader = "Pop. by Quarter";
+    const bodyHeader = entities.map((entity: AnyObject) =>
+      entity.transitionBenchmarks_targetPopulationName_short
+        ? entity.transitionBenchmarks_targetPopulationName_short
+        : truncateHeader(entity.transitionBenchmarks_targetPopulationName)
```

Comment:
```suggestion
entity.transitionBenchmarks_targetPopulationName_short ?? truncateHeader(entity.transitionBenchmarks_targetPopulationName)
```

can simplify with nullish coalescing